### DISCUSSION TITLE
Can no longer reach the security onion console after the update 2.3.90 -2.3.100

### DISCUSSION THREAD

**Discussion by cozazura** [Security-Onion-Solutions/securityonion#7161] [Unsupported Versions] [answered]
After the update when i reboot the manager node i can acces the interface for about 10 mins while it's proming the message "Disconnected from manager " "attempting to reconnect". After that if i refresh the page i get the classic "This site can't be reached"  . I only have 2 nodes : the manager and the search one ...they are both visible and so-status shows that everything is ok . I have no ideea what to do .
edit:
it is installed on a ubuntu 20.04 server 
i had no failure when i ran "sudo salt-call state.highstate"

**Comment by dougburks** [comment, marked as answer]
Can you ping the manager?

What browser are you using?

Have you tried shift-reload in your browser?

Are there any network firewalls or other network devices between you and the manager?

Have you previously run `sudo so-allow` to allow your traffic through the host firewall?

Are you running Wazuh? Have you verified that `so-allow` added your IP address to the Wazuh white_list in `/nsm/wazuh/etc/ossec.conf`?

**Reply by cozazura** [reply]
yes i can ping it 
i use chrome 
yes i did 
no
i did run so-allow
i don't use wazuh
it's odd because now it fixed itself ...everything working fine
